### PR TITLE
Adding sample of descending property to OrderBy Javadoc

### PR DESCRIPTION
**Pull request by mswatosh**
I noticed the class javadoc for `@OrderBy` doesn't mention or show an example of the `descending` property. I'm not sure if it's the case for other IDE's but in VSCode, the javadoc shown when hovering over an annotation doesn't include the properties, so without a description in the class javadoc, the availability of the `descending` property may not be obvious.

### CODE REVIEW

**Review by njr-11 — APPROVED** [review]
Thanks for contributing the improvement!

File: api/src/main/java/jakarta/data/repository/OrderBy.java [hunk 1]
```diff
@@ -58,6 +58,13 @@
  * <li>ascending order for boolean values places {@code false} before
  *     {@code true}.</li>
  * </ul>
+ * 
+ * The default sort is ascending. The {@code descending} property can be
```
Comment (gavinking) [inline api/src/main/java/jakarta/data/repository/OrderBy.java:62]:
```suggestion
 * The default sort order is ascending. The {@code descending} member can be
```

File: api/src/main/java/jakarta/data/repository/OrderBy.java [hunk 2]
```diff
@@ -58,6 +58,13 @@
...
+ * used to specify the sort direction.
```
Comment (gavinking) [inline api/src/main/java/jakarta/data/repository/OrderBy.java:63]:
```suggestion
 * used to specify the sort direction.</p>
```

**Review by gavinking — APPROVED** [review]
Thanks.

**Comment by mswatosh** [comment]
The build didn't like the `</p>` change, but I think I understand what you were trying to fix and corrected it here: [06ffea1](https://github.com/jakartaee/data/pull/877/commits/06ffea14a8ce9dec0757c3a3be7d638664a15157)

**Pull request merged**